### PR TITLE
Staging

### PR DESCRIPTION
**Pull request by tholulomo**
closes #225

### CODE REVIEW

**Review by aswallace — APPROVED** [review]

File: app/src/assets/css/modules/_utility.scss [hunk 1]
```diff
@@ -39,10 +39,34 @@
     }
 
     &_divider {
-        height: .1rem;
+        // height: .1rem;
+        height: .2rem; //TODO-check 06/16/2022
```
Comment (aswallace) [inline app/src/assets/css/modules/_utility.scss:43]: Same with these

File: app/src/pages/explorer/parameterized-query/parameterized-query-page.vue [hunk 2]
```diff
@@ -171,6 +172,11 @@ export default {
     },
     newQuery () {
       return this.query !== this.lastRunQuery
+    },
+    pageTitle () {
+      const selectionsKeys = Object.keys(this.varSelections)
+      const title = this.varSelections[selectionsKeys[3]]
+      return title[0].toUpperCase() + title.substring(1)
```
Comment (aswallace) [inline app/src/pages/explorer/parameterized-query/parameterized-query-page.vue:179]: I'm running into some issues with this change locally. It works fine for me for the first two queries but gives errors on some of the others that have fewer selection keys

**Comment by tholulomo** [comment]
Implemented the needed additional test for the new UI implementations

File: app/src/pages/explorer/parameterized-query/parameterized-query-page.vue (hunk 2 above)
Comment (tholulomo) [reply app/src/pages/explorer/parameterized-query/parameterized-query-page.vue]: @aswallace please pull the changes again and we can sync up to discuss

**Pull request merged**